src/Modal.js

```diff
@@ -162,9 +166,34 @@ class Modal extends React.Component {
       }
     }
 
+    document.removeEventListener('focus', this.trapFocus);
```

Comment:
```suggestion
    document.removeEventListener('focus', this.trapFocus, true);
```

Reply:
Sorry missed this one.  Need to add `true` here since you added the original handler using capture.

Reply:
Hey @kyletsang, just included this change and included a test case for assuring the listeners are added/removed while mounting and unmounting the component.